custom_documentation/endpoint/data_stream/file/linux/linux_file_delete.yaml

```
@@ -0,0 +1,16 @@
+overview:
+  name: Linux File Delete
+  description: >
+    This event is generated when a file is deleted.
+identification:
+  filter:
+    event.action: deletion
```

Is there any performance benefit to filtering on the standard categorization fields that have fixed values?
```suggestion
    event.type: deletion
```